backend/app/models/licenses.py

```diff
@@ -13,20 +14,18 @@ class LicenseBase(BaseModel):
     text: str
     url: str
     version: str
-    holders: List[UserOut]
-    expiration_date: datetime = Field(default_factory=datetime.utcnow)
+    holders: List[UserOut] = []
```

Comment:
I think this is the key change. I wonder why this creates issue in initializing the class, when no other variables are initialized.